### ISSUE TITLE
Unknown word in mixins

### ISSUE THREAD

**Issue by rodoabad** [stylelint/stylelint#1151]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

I'm getting an unknown word error when I'm trying to lint a sass file that has the following mixin.

`_icons.scss:5:13: Unknown word`

> What CSS is needed to reproduce this issue?

```scss
@mixin ui-svg-icon($svg, $height: 1, $width: 1) {

    .icon-#{$svg},
    .icon-#{$svg}-small,
    .icon-#{$svg}-medium{
        background-image: url('#{$icons-path}/#{$svg}.svg');
        background-repeat: no-repeat;
        display: inline-block;
        vertical-align: middle;
    }

    .icon-#{$svg} {
        height: $height;
        width: $width;
    }

    .icon-#{$svg}-small{
        width: $icon-size-small !important;
        height: ($height * $icon-size-small) / $width !important;
    }

    .icon-#{$svg}-medium{
        width: $icon-size-medium !important;
        height: ($height * $icon-size-medium) / $width !important;
    }

}
```

> Which version of stylelint are you using?

`5.4.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`"stylelint ./lib/**/*.scss ./src/**/*.scss"`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS mixins

> What did you expect to happen?

No warnings to be flagged if I set my PostCSS parser to use `postcss-scss`.

**Comment by evilebottnawi** [comment]
@rodoabad 
Change `.icon-#{$svg}-medium{` -> `.icon-#{$svg}-medium {`
Working or not?

**Comment by evilebottnawi** [comment]
@rodoabad oh sorry, you not right run, use `stylelint --syntax scss ./lib/**/*.scss ./src/**/*.scss`

**Comment by rodoabad** [comment]
Aah thanks, @evilebottnawi. For anyone else that's having this trouble they have it in their docs - https://github.com/stylelint/stylelint/blob/master/docs/user-guide/cli.md#usage-examples
